packages/local-refresh/src/actions/CopyAppDistFiles.ts

```diff
@@ -51,22 +53,26 @@ export default class ResetDatabase implements ActionInterface {
       });
     }
 
-    const { files } = await prompts({
-      type: "multiselect",
-      name: "files",
-      message: "Which config files do you want to copy?",
-      choices: Array.from(appConfigDistFiles.keys()).map((file) => ({
-        title: appConfigDistFiles.get(file) || file,
-        value: file,
-      })),
-      hint: "- Space to select. Return to submit",
-    });
+    if (!noInteraction) {
+      const { files } = await prompts({
+        type: "multiselect",
+        name: "files",
+        message: "Which config files do you want to copy?",
+        choices: Array.from(appConfigDistFiles.keys()).map((file) => ({
+          title: appConfigDistFiles.get(file) || file,
+          value: file,
+        })),
+        hint: "- Space to select. Return to submit",
+      });
+
+      if (!files) {
+        return false;
+      }
 
-    if (!files) {
-      return false;
+      this.filesToCopy = files;
     }
 
-    this.filesToCopy = files;
+    this.filesToCopy = Array.from(appConfigDistFiles.keys());
```

Comment:
This would break the interactive version. Let's move the non-interactive version conditional to early exit after we have a list of files. 

Something like this?

```ts
// ...
for (const dir of phpAppDirectories) {
  // ...
}

// ...
if (noInteraction) {
  this.filesToCopy = Array.from(appConfigDistFiles.keys());
}

// ...
const { files } = await prompts({
  // ...
});
// ...
```